nrf_wifi/fw_if/umac_if/inc/fmac_api_common.h

```diff
@@ -362,6 +362,17 @@ enum nrf_wifi_status nrf_wifi_fmac_set_packet_filter(void *dev_ctx, unsigned cha
 						     unsigned short buffer_size);
 #endif /* CONFIG_NRF700X_RAW_DATA_RX || CONFIG_NRF700X_PROMISC_DATA_RX */
 
+/**
+ * @brief Issue a request to reset stats of the RPU.
+ * @param fmac_dev_ctx Pointer to the UMAC IF context for a RPU WLAN device.
+ *
+ * This function is used to send a command to
+ *          instruct the firmware to reset current RPU statistics.
```

Comment:
Why blank space before `instruct` ?

Reply:
I have seen similar format in file. Removed now.